src/DependencyInjection/DI.External.Tests/test/Microsoft.Extensions.DependencyInjection.ExternalContainers.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <RootNamespace>Microsoft.Extensions.DependencyInjection</RootNamespace>
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Newtonsoft.Json" />
+
+    <Reference Include="Microsoft.Extensions.DependencyInjection" />
+    <ProjectReference Include="..\..\DI.Specification.Tests\src\Microsoft.Extensions.DependencyInjection.Specification.Tests.csproj" />
```

Comment:
```suggestion
    <Reference Include="Microsoft.Extensions.DependencyInjection.Specification.Tests" />
```

One more

Reply:
Oh, so we do it for tests projects too?

Reply:
You're right about using ProjectRef to other test projects, but Microsoft.Extensions.DependencyInjection.Specification.Tests is actually a class library of tests.